toolchest_client/tools/tool.py

```diff
@@ -289,9 +289,8 @@ def _generate_jobs(self, should_run_in_parallel):
         else:
             # Make sure we're below plan/multi-part limit for non-splittable files
             for file_path, file_is_in_s3 in zip(self.input_files, self.inputs_are_in_s3):
-                if not file_is_in_s3:
-                    check_file_size(file_path, max_size_bytes=FOUR_POINT_FIVE_GIGABYTES)
-                # If the file is already in S3, no need to check max size.
+                check_file_size(file_path, max_size_bytes=FOUR_POINT_FIVE_GIGABYTES, is_s3_uri=file_is_in_s3)
```

Comment:
Nit: `is_s3_uri` and `file_is_in_s3` represent the same thing but have different names